[2024-07-24T13:30:17.795+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-24T13:30:17.804+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_hooks.pred_time_series scheduled__2024-07-23T13:30:00+00:00 [queued]>
[2024-07-24T13:30:17.807+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_hooks.pred_time_series scheduled__2024-07-23T13:30:00+00:00 [queued]>
[2024-07-24T13:30:17.807+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-07-24T13:30:17.812+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): pred_time_series> on 2024-07-23 13:30:00+00:00
[2024-07-24T13:30:17.816+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62: DeprecationWarning: This process (pid=4806) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-07-24T13:30:17.817+0000] {standard_task_runner.py:64} INFO - Started process 4817 to run task
[2024-07-24T13:30:17.818+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_hooks', 'pred_time_series', 'scheduled__2024-07-23T13:30:00+00:00', '--job-id', '611', '--raw', '--subdir', 'DAGS_FOLDER/Stock_pred.py', '--cfg-path', '/tmp/tmpq35w1rdw']
[2024-07-24T13:30:17.818+0000] {standard_task_runner.py:91} INFO - Job 611: Subtask pred_time_series
[2024-07-24T13:30:17.837+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_with_postgres_hooks.pred_time_series scheduled__2024-07-23T13:30:00+00:00 [running]> on host b6b4a3e7bf9d
[2024-07-24T13:30:17.870+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='***@example.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_with_postgres_hooks' AIRFLOW_CTX_TASK_ID='pred_time_series' AIRFLOW_CTX_EXECUTION_DATE='2024-07-23T13:30:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-07-23T13:30:00+00:00'
[2024-07-24T13:30:17.870+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-24T13:30:17.951+0000] {warnings.py:112} WARNING - /opt/***/dags/Stock_pred.py:262: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  df_new_ts.rename(index={0:date},inplace=True)

[2024-07-24T13:30:17.957+0000] {base.py:84} INFO - Using connection ID 'postgres_server_2' for task execution.
[2024-07-24T13:30:18.050+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/keras/src/saving/saving_lib.py:576: UserWarning: Skipping variable loading for optimizer 'rmsprop', because it has 7 variables whereas the saved optimizer has 12 variables. 
  saveable.load_own_variables(weights_store.get(inner_path))

[2024-07-24T13:30:18.154+0000] {logging_mixin.py:188} INFO - 1/1 ━━━━━━━━━━━━━━━━━━━━ 0s 76ms/step
[2024-07-24T13:30:18.154+0000] {logging_mixin.py:188} INFO - 1/1 ━━━━━━━━━━━━━━━━━━━━ 0s 77ms/step
[2024-07-24T13:30:18.161+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-07-24T13:30:18.161+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-24T13:30:18.164+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_hooks, task_id=pred_time_series, run_id=scheduled__2024-07-23T13:30:00+00:00, execution_date=20240723T133000, start_date=20240724T133017, end_date=20240724T133018
[2024-07-24T13:30:18.208+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-07-24T13:30:18.219+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-07-24T13:30:18.220+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
